src/style_manager/view/PropertyView.js

```diff
@@ -427,6 +439,37 @@ module.exports = Backbone.View.extend({
           (stylableReq.indexOf(id) >= 0 || stylableReq.indexOf(property) >= 0));
     }
 
+    // Check if the property is available based on other property's values
+    if (sectors && requires) {
+      const properties = Object.keys(requires);
+      sectors.each(sector => {
+        sector.get('properties').each(model => {
+          if (properties.includes(model.id)) {
+            const values = requires[model.id];
+            stylable = stylable && values.includes(model.get('value'));
+          }
+        });
+      });
+    }
+
+    // Check if the property is available based on parent's property values
+    if (requiresParent) {
+      if (
+        trg.view &&
+        trg.view.$el &&
+        trg.view.$el[0] &&
+        trg.view.$el[0].parentNode
+      ) {
+        const styles = window.getComputedStyle(trg.view.$el[0].parentNode);
+        Object.entries(requiresParent).forEach(([property, values]) => {
```

Comment:
`Object.entries` is not available in IE, use `_.each(requiresParent, (values, property) => {...})`